src/api-service/__app__/onefuzzlib/orm.py

```diff
@@ -66,6 +67,26 @@
 HOURS = 60 * 60
 
 
+class HasState(Protocol):
+    # TODO: this should be bound tighter than Any
```

Comment:
Could you document that, in particular, we expect `state` to have a `name` property, so it must not be a bare `str`?

Incidentally, can we at least say it must an `Enum`?